### ISSUE TITLE
Run in background once the connection is established

### ISSUE THREAD

**Issue by AaronMcHale** [adrienverge/openfortivpn#885]
Hello

My use case is I open a terminal window, run openfortivpn, enter my password, when the connection is established I'd like the process to release stdin and fork to the background. That way I can continue to use the terminal window and don't need a terminal open specifically for the VPN connection. When I'm done, I would want to close the terminal window and have the VPN connection close.

I initially tried just running the process in the background, but it needs to be in the foreground initially so I can supply my credentials.

Basically, if thee was a flag that would allow the process to release stdin and return me to my prompt once the connection was established, but still output to stderr if an issue occurs (e.g. connection drops), then that would be idea.

Thanks,
-Aaron

**Comment by DimitriPapadopoulos** [comment]
See #229 and #594 among others.

**Comment by AaronMcHale** [comment]
I did see both of those before logging this issue, but they are slightly different in that their use case is for running as a daemon, whereas what I'm saying is, run in foreground initially, but then once connection is established release stdin and return the prompt, but continue to log to stdout/stderr if need be.

**Comment by mrbaseman** [comment]
what about running _openfortivpn_ inside of a `screen` session from which you detach once the connection is up. When it freezes you can re-attach and check for possible error messages
